website/blog/2018-12-21-state-of-react-native-community.md

```diff
@@ -0,0 +1,51 @@
+---
+title: The State the React Native Community in 2018
+author: Lorenzo Sciandra
+authorTitle: Core Maintainer & React Native Developer
+authorURL: https://github.com/kelset
+authorImageURL: https://avatars2.githubusercontent.com/u/16104054?s=460&v=4
+authorTwitter: kelset
+category: announcements
+---
+
+Two years from now, we’ll probably look back at 2018 as a turning point in the React Native development history.
+
+A lot of eyes are currently pointed at the incoming rewrite of the architecture, widely known as [Fabric](https://github.com/react-native-community/discussions-and-proposals/issues/4), which will enable our mobile apps to reach new peaks of mobile performances (alongside JSI & TurboModules, which are equally as important to reach said goal).
```

Comment:
"reach new peaks of mobile performances" sounds a bit wrong, because you want low performance. Fabric is also not only there to fix performance problems, but other fundamental flaws in RN's architecture. I would probably recommend rewording this paragraph like this:

> A lot of people are excited about the rewrite of React Native's architecture, widely known as [Fabric](https://github.com/react-native-community/discussions-and-proposals/issues/4). Among other things, this will fix fundamental flaws in React Native's architecture and will set up React Native for success in the future together with JSI and TurboModules (+ add links pls).

Reply:
I’m not sure I understand here - low performance is the goal? Isn’t ‘high’ the right adjective? I’ll ask around.

Reply:
The suggested wording is 👌, but “low performance” means “suboptimal”

Reply:
haha probably just a mistakes XD
i'd never want low performance

Reply:
Oops, yeah you are right. Sorry I had a baby crawling around me while I wrote this! :D

I meant you want lower wait times but obviously higher performance. Peak performance works but I think “new peaks of performance” sounds off.